sklearn/utils/_repr_html/params.py

```diff
@@ -2,11 +2,36 @@
 # SPDX-License-Identifier: BSD-3-Clause
 
 import html
+import re
 import reprlib
 from collections import UserDict
+from urllib.parse import quote
 
 from sklearn.utils._repr_html.base import ReprHTMLMixin
 
+CLASS_DOC_URL_PREFIX = "https://scikit-learn.org/{doc_version}/modules/generated/"
+
+
+def link_to_param_doc(estimator_class, param_name, doc_link):
+    """URL to the relevant section of the docstring using a Text Fragment
```

Comment:
Let's write a small unit test for this function in `test_params.py` as for over private helper.